docs/book/src/reference/providers.md

```diff
@@ -34,6 +34,7 @@ updated info about which API version they are supporting.
 - [VMware Cloud Director](https://github.com/vmware/cluster-api-provider-cloud-director)  
 - [vSphere](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere)
 - [Microvm](https://github.com/weaveworks-liquidmetal/cluster-api-provider-microvm)
+- [Virtink](https://github.com/smartxworks/cluster-api-provider-virtink)
```

Comment:
do you mind sorting the list in alphabetical order, to make things easier for users
otherwise I can take care of this in a follow up

Reply:
I'm glad to do this.